dxp/exporting.md

```diff
@@ -1,47 +1,61 @@
 # Exporting your JavaScript
 
-This document describes how you should organize you npm packages to export JavaScript functionality
+This document describes how you should organize you OSGi modules to export JavaScript functionality.
```

Comment:
"you OSGi modules" → "your OSGi modules"

Reply:
Thanks!